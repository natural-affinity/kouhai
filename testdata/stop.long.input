kouhai --stop "fake-exe-path"
