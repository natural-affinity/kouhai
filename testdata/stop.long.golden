main.go:53: sh: fake-exe-path: command not found

